perception/autoware_traffic_light_classifier/src/traffic_light_classifier_node.cpp

```diff
@@ -84,6 +84,9 @@ void TrafficLightClassifierNodelet::imageRoiCallback(
   if (classifier_ptr_.use_count() == 0) {
     return;
   }
+  if (input_rois_msg->rois.size() == 0) {
```

Comment:
It's a minor suggestion, but this one is easier to read.
```suggestion
  if (input_rois_msg->rois.empty()) {
```

Reply:
resolved https://github.com/autowarefoundation/autoware.universe/pull/7608/commits/aec7e9743f0fb9381c470415eaae4602dc5419b2